app/src/app/api/v0/city/[city]/user/[user]/route.ts

```diff
@@ -0,0 +1,58 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import { createCityRequest, createUserRequest } from "@/util/validation";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+
+export const GET = apiHandler(async (_req: NextRequest, { params }) => {
```

Comment:
Are we sure the user routes need to be nested under the `city/[city]` folder? It could be `api/v0/user/[user]` instead, which is shorter and it seems like the city isn't really necessary here...

Reply:
right right, i was actually debating. Ill resolve it